src/connect_async.rs

```diff
@@ -0,0 +1,189 @@
+use std::io::{self, Read, Write};
+
+use futures::{Poll, Future, Async};
+use native_tls;
+use native_tls::{HandshakeError, Error, TlsConnector, TlsAcceptor};
+#[allow(deprecated)]
+use tokio_core::io::Io;
```

Comment:
This import isn't need in reqwest, it was only in tokio-tls for backwards compatibility.